core/templates/pages/error-pages/error-page-root.component.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Root component for error page.
+ */
+
+import { Component } from '@angular/core';
+import { WindowRef } from 'services/contextual/window-ref.service';
+import { PageTitleService } from 'services/page-title.service';
+
+@Component({
+  selector: 'oppia-error-page-root',
```

Comment:
Is this componenet going to be used for all other error pages except 404?

Reply:
yeah

Reply:
Do we have a code comment in this file stating that this page is not use for 404 error page?

Reply:
done